_1327/documents/templates/documents_edit.html

```diff
@@ -602,7 +622,125 @@ <h4 class="modal-title" id="deleteAutosaveHeader">{% trans "Do you really want t
+                    name: "GeneralTextBlockGroup",
+                    data: [{
+                        name: "cmdAddStartText",
+                        toggle: false,
+                        title: "Begin or continue the meeting",
+                        icon: "glyphicon glyphicon-play",
+						callback: function (e) {
+                            const markdownEditor = e.$editor.find(".md-input");
+
+							const currentTime = getCurrentTime();
+							let textToInsert;
+
+							const unfinishedBreak = markdownEditor[0].value.match(/\|break\|\((\d+):(\d+)\)\(\)/);
+							const startedMeeting = markdownEditor[0].value.search(/\|start\|\((\d+):(\d+)\)/);
+
+							if(unfinishedBreak) {
+								const unfinishedBreakPosition = markdownEditor[0].value.search(/\|break\|\((\d+):(\d+)\)\(\)/);
+								const relativeInsertPosition = unfinishedBreak[0].search(/\(\)/) + 1;
+								
+								const oldSelectionStartPosition = markdownEditor[0].selectionStart;
+								const oldSelectionEndPosition = markdownEditor[0].selectionEnd;
+								
+								// set cursor position to absolute insert position
+								markdownEditor[0].selectionStart = unfinishedBreakPosition + relativeInsertPosition;
+								markdownEditor[0].selectionEnd = unfinishedBreakPosition + relativeInsertPosition;
+								const currentSelectionStartPosition = markdownEditor[0].selectionStart;
+
+								textToInsert = currentTime;
+								insertText(textToInsert, markdownEditor);
+
+								// cursor shall stay at its old (relative) position
+								markdownEditor[0].selectionStart = oldSelectionStartPosition;
+								markdownEditor[0].selectionEnd = oldSelectionEndPosition;
+								if (oldSelectionStartPosition > currentSelectionStartPosition) {	
+									markdownEditor[0].selectionStart += textToInsert.length;
+									markdownEditor[0].selectionEnd += textToInsert.length;
+								}
+
+							} else if (startedMeeting < 0) { // meeting only needs to be started once
+								textToInsert = "|start|(" + currentTime + ")";
+								insertText(textToInsert, markdownEditor);
+							}	
+                        }
+					},
+					{
+                        name: "cmdAddBreakText",
+                        toggle: false,
+                        title: "Pause the meeting",
+                        icon: "glyphicon glyphicon-pause",
+						callback: function (e) {
+							const markdownEditor = e.$editor.find(".md-input");
+
+							const currentTime = getCurrentTime();
+							const textToInsert = "\n|break|(" + currentTime + ")()";
+							
+							insertText(textToInsert, markdownEditor);
+						}
+					},
+					{
+                        name: "cmdAddEndText",
+                        toggle: false,
+                        title: "End the meeting",
+                        icon: "glyphicon glyphicon-stop",
+						callback: function (e) {
+                            const markdownEditor = e.$editor.find(".md-input");
+							
+							const currentTime = getCurrentTime();
```

Comment:
The indentation is weird here as well as in some other places in this file. I'm pretty sure there's an easy way to enable auto-formatting in most editors.
For example, in VS Code on Windows, in the settings (<kbd>Ctrl</kbd> + <kbd>,</kbd>) you can enable "Editor: Format On Save" (although you probably need to have a HTML/JS language extension installed for that to work).